packages/docs/src/components/Navbar/NavbarContent/styles.module.css

```diff
@@ -0,0 +1,156 @@
+.navbar_container {
```

Comment:
here you switched from camelCase to snake_case. for consistency please stick to one type of casing. i think we use camelCasing for styling